specification/communication/data-plane/JobRouter/stable/2023-11-01/communicationservicejobrouter.json

```diff
@@ -0,0 +1,4502 @@
+            "$ref": "./examples/Offers_DeclineJobOffer.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AcceptJobOfferResult": {
+      "type": "object",
+      "description": "Response containing Id's for the worker, job, and assignment from an accepted offer",
+      "properties": {
+        "assignmentId": {
+          "type": "string",
+          "description": "The assignment Id that assigns a worker that has accepted an offer to a job."
+        },
+        "jobId": {
+          "type": "string",
+          "description": "The Id of the job assigned."
+        },
+        "workerId": {
+          "type": "string",
+          "description": "The Id of the worker that has been assigned this job."
+        }
+      },
+      "required": [
+        "assignmentId",
+        "jobId",
+        "workerId"
+      ]
+    },
+    "Azure.Core.Foundations.Error": {
+      "type": "object",
+      "description": "The error object.",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "One of a server-defined set of error codes."
+        },
+        "message": {
+          "type": "string",
+          "description": "A human-readable representation of the error."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the error."
+        },
+        "details": {
+          "type": "array",
+          "description": "An array of details about specific errors that led to this reported error.",
+          "items": {
+            "$ref": "#/definitions/Azure.Core.Foundations.Error"
+          },
+          "x-ms-identifiers": []
+        },
+        "innererror": {
+          "$ref": "#/definitions/Azure.Core.Foundations.InnerError",
+          "description": "An object containing more specific information than the current object about the error."
+        }
+      },
+      "required": [
+        "code",
+        "message"
+      ]
+    },
+    "Azure.Core.Foundations.ErrorResponse": {
+      "type": "object",
+      "description": "A response containing error details.",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/Azure.Core.Foundations.Error",
+          "description": "The error object."
+        }
+      },
+      "required": [
+        "error"
+      ]
+    },
+    "Azure.Core.Foundations.InnerError": {
+      "type": "object",
+      "description": "An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses.",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "One of a server-defined set of error codes."
+        },
+        "innererror": {
+          "$ref": "#/definitions/Azure.Core.Foundations.InnerError",
+          "description": "Inner error."
+        }
+      }
+    },
+    "Azure.Core.eTag": {
```

Comment:
This snippet is used as an example in our suppression guide reference at https://aka.ms/pr-suppressions. Adding this comment here for easy hyperlinking.